### ISSUE TITLE
Files should be deleted from \files folder too if i delete them from "Dateiliste"

### ISSUE THREAD

**Issue by pirol1972** [giowck/symphytum#104]
if i delete the files from \files via program or if i delete them from disk via file manager what should the program do? if i delete them with both variantes the program shows an error if i made a backup the files were not found.

**Comment by joshirio** [comment]
Hi when you import a file into Symphytum's database the software takes the ownership of the file and stores it in the files directory in the corresponding configuration path, where users are not expected to access and modify/delete files on their own. Symphytum takes care of deleting the files when no longer needed (especially the menu "Tools->Database->Clean unused files" should clean the files directory). So, if you don't touch the files directory manually no issues should appear, unless there's a bug, which I wasn't able to reproduce by testing. If you believe so, please try to explain extact steps to reproduce it.

So in short: always use Symphytum to manage files, never delete manually or else the software won't be able to update the metadata correctly.

Thanks, also for the reports you send me via mail, I'll look into it.
